tests/lean/run/PPTopDownAnalyze.lean

```diff
@@ -225,6 +225,13 @@ set_option pp.analyze.typeAscriptions false in
 #testDelab (fun (x : Unit) => @id (ReaderT Bool IO Bool) (do read : ReaderT Bool IO Bool)) ()
   expecting (fun (x : Unit) => id read) ()
 
+instance : CoeFun Bool (fun b => Bool → Bool) := { coe := fun b x => b && x }
+set_option pp.analyze.trustCoe false in
+#testDelab coeFun true false expecting coeFun (γ := fun b => Bool → Bool) true false
```

Comment:
The Lean 4 elaborator doesn't produce coe/coeFun/coeSort anymore.  Does it even make sense to add special support for these functions to the delaborator then?

Reply:
I was wondering this as well. I was just trying to be consistent with the existing tests/behavior for now. Perhaps there is some use-case where users manually add `coeFun` but then don't want it displayed as such?

Reply:
Yeah, I think I was still thinking of Lean 3 when I added them